src/components/Company/CompanyName/CorrectNameRequest.vue

```diff
@@ -113,7 +113,7 @@ export default class CorrectNameRequest extends Mixins(NameRequestMixin) {
       .then(response => {
         const nrCorrection: NrCorrectionIF = {
           nrNumber: this.nameRequestNumber,
-          legalName: this.getApprovedName(response)
+          legalName: this.getNrApprovedName(response)
```

Comment:
Beat me too it :P (Still in my local), thanks!

Reply:
I did this to get rid of errors when running locally.
I will commit this fix separately so **this** PR is only the LD stuff (unless you beat me to committing a fix ;) ).

Reply:
Separate PR has been created, so this conversation is now obsolete.